kafka/resource_kafka_topic.go

```diff
@@ -86,7 +86,7 @@ func topicCreate(d *schema.ResourceData, meta interface{}) error {
 
 func topicCreateFunc(client *LazyClient, t Topic) resource.StateRefreshFunc {
 	return func() (result interface{}, s string, err error) {
-		topic, err := client.ReadTopic(t.Name)
+		topic, err := client.ReadTopic(t.Name, true)
```

Comment:
anything that potentially deals with modified cluster state should refresh the metadata.